compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/ScenarioServiceMain.scala

```diff
@@ -292,8 +293,10 @@ class ScenarioService(implicit
 
         } catch {
           case e: archive.Error =>
+            // TODO error codes: Adapt V2 ?
```

Comment:
Language team

Reply:
Scenarios will die completely in Daml 2.0. 
Safe to ignore.